src/components/itemContextMenu.js

```diff
@@ -11,6 +11,20 @@ import ServerConnections from './ServerConnections';
 import toast from './toast/toast';
 import * as userSettings from '../scripts/settings/userSettings';
 
+function getDeleteCommandName(type) {
```

Comment:
nit: i believe "label" is usually the terminology used for webapp text elements

```suggestion
function getDeleteLabel(type) {
```

Reply:
Sure, Updated